### PR TITLE
add iperf microservice based on the individual iperf bundle

### PR DESCRIPTION
**Pull request by ewanglong**

### CODE REVIEW

File: iperf/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# iperf3 service on clearlinux
+Iperf is a widely used tool for network performance measurement and tuning.
+
+## Build
+```
+sudo docker build -t clr-iperf:latest .
```
Comment (phmccarty) [inline iperf/README.md]: Should the name of the image be `clearlinux/iperf` instead?

File: iperf/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+```
+
+## Run as Server:
+```
+sudo docker run -it --rm --name=iperf-srv -p 5201:5201 clr-iperf -s
```
Comment (gtkramer) [inline iperf/README.md]: Same for the image name here.  Probably we want `clearlinux/iperf`?

File: iperf/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+```
+
+## Run as Client (first get server IP address):
+```
+sudo docker run -it --rm --network=host clr-iperf -c <SERVER_IP>
```
Comment (gtkramer) [inline iperf/README.md]: Ditto for image name

File: iperf/README.md (hunk 2 above)
Comment (phmccarty) [reply iperf/README.md]: Yes, I think `clearlinux/iperf` is the expected name.

File: iperf/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
+FROM clearlinux:latest
+MAINTAINER long1.wang@intel.com
+
+ARG swupd_args
+
+RUN swupd update $swupd_args && \
```
Comment (qzheng527) [inline iperf/Dockerfile]: Please add "--no-boot-update" as parameter to "swupd update".
The update on container doesn't need update the boot partition.

**Comment by ewanglong** [comment]
close this and will submit new PR.

**Pull request closed without merging**